bolt04/enctlvs.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "test name": "Simple enctlv for Alice, next is Bob",
+    "node_privkey": "414141414141414141414141414141414141414141414141414141414141414101",
+    "node_id": "02eec7245d6b7d2ccb30380bfbe2a3648cd7a942653f5aa340edcea1f283686619",
+    "blinding_secret": "050505050505050505050505050505050505050505050505050505050505050501",
```

Comment:
@rustyrussell I'm wondering why this is 33 bytes? Aren't secrets always 32 bytes?

Reply:
It's the bitcoin-style notation for private keys.... append 01.  Horrible, let's kill it...